imports/plugins/included/shipping/server/methods.js

```diff
@@ -1,19 +1,26 @@
 import { Shipping } from "/lib/collections";
 import { Reaction } from "/server/api";
 
-Meteor.methods({
-  /*
+export const methods = {
+  /**
```

Comment:
I guess that it make sense that permissions are included in the wrapper (I've been thinking for a while that we could just extend the Meteor.method with our own Reaction.method and just include the permissions etc at that level.  You have a link to your implementation?